packages/amplify_core/pubspec.yaml

```diff
@@ -7,12 +7,14 @@ environment:
   sdk: ">=2.17.0 <3.0.0"
 
 dependencies:
+  ansicolor: ^2.0.1
   async: ^2.8.2
   aws_common: ^0.1.0
   aws_signature_v4: ^0.1.0
   collection: ^1.15.0
   intl: ^0.17.0
   json_annotation: ^4.4.0
+  logging: ^1.0.2
```

Comment:
```suggestion
  logging: ^1.0.0
```

Reply:
Why would we use an older version of logging?

Reply:
This widens the acceptable range from "all versions above 1.0.2 but below 2.0.0" to "all versions between 1.0.0 and 2.0.0". There's not much benefit to us but it allows for people to use our library who have `logging: 1.0.0` in their pubspec without adding a dependency override and we don't particularly care if we resolve to 1.0.0 since there's nothing in 1.0.2 we need.